Fwk/AppFwk/cafUserInterface/cafPdmUiTableViewEditor.cpp

```diff
@@ -411,7 +413,24 @@ bool PdmUiTableViewEditor::eventFilter( QObject* obj, QEvent* event )
     {
         this->updateSelectionManagerFromTableSelection();
     }
-    // standard event processing
+
+    auto keyEvent = dynamic_cast<QKeyEvent*>( event );
+    if ( keyEvent && keyEvent->matches( QKeySequence::Copy ) )
+    {
+        QString             text;
+        QItemSelectionRange range = m_tableView->selectionModel()->selection().first();
```

Comment:
You might want to iterate over all selections here, not just the first one, to handle cases where the user uses ctrl-click to select non-contiguous item ranges.

Reply:
Fixed